apps/meteor/app/livechat/server/api/v1/room.ts

```diff
@@ -77,6 +77,7 @@ API.v1.addRoute(
 				const roomInfo = {
 					source: {
 						type: isWidget(this.request.headers) ? OmnichannelSourceType.WIDGET : OmnichannelSourceType.API,
+						destination: isWidget(this.request.headers) ? this.request.headers.host : undefined,
```

Comment:
Maybe use the conditional only once and have separate objects for Widget and API?
```ts
				const roomInfo = {
					source: {
						...(isWidget(this.request.headers)
							? { type: OmnichannelSourceType.WIDGET, destination: this.request.headers.host }
							: { type: OmnichannelSourceType.API }),
					},
				};
```